sycl/include/sycl/queue.hpp

```diff
@@ -245,6 +249,184 @@ class __SYCL_EXPORT queue : public detail::OwnerLessBase<queue> {
   queue(const context &SyclContext, const device &SyclDevice,
         const async_handler &AsyncHandler, const property_list &PropList = {});
 
+#else // __SYCL_EXT_ONEAPI_BACKEND_LEVEL_ZERO_V3
+
+private:
+  // This class is used as an additional internal parameter to distinguish older
+  // constructors from current ones.
+  class Discriminator {
+  public:
+    Discriminator(){};
+  };
+
+public:
+  /// Constructs a SYCL queue instance using the device returned by an instance
+  /// of default_selector.
+  ///
+  /// \param PropList is a list of properties for queue construction.
+  explicit queue(const property_list &PropList = {}, Discriminator Disc = {})
+      : queue(default_selector(), detail::defaultAsyncHandler, PropList, Disc) {
+  }
```

Comment:
are we really allowed to add non-standard members?

Reply:
Can we have the summary of the offline discussion posted here?